question1.java

```diff
@@ -0,0 +1,136 @@
+package question1;
+
+import java.io.BufferedReader;
+import java.io.FileNotFoundException;
+import java.io.FileReader;
+import java.io.IOException;
+import java.io.InputStreamReader;
+import java.util.StringTokenizer;
+
+/**
+ * Custom Scanner which reads lines from a file and parses the information

```

Comment:
I meant if the user expected t read an Integer, they would use the nextInt method and so on.